sysfs-led-main.c

```diff
@@ -638,12 +711,24 @@ static void
 sysfs_led_generate_ramp(int ms_on, int ms_off)
 {
   switch( led_control_breath_type(&led_control) ) {
+  case LED_RAMP_SINE:
+    sysfs_led_generate_ramp_sine(ms_on, ms_off, false);
+    break;
+
+  case LED_RAMP_SAWTOOTH:
+      sysfs_led_generate_ramp_sawtooth(ms_on, ms_off);
```

Comment:
This and the next case have indentation mismatch to other cases.

Reply:
fixed